instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpServerAttributesExtractor.java

```diff
@@ -75,14 +75,7 @@ public static <REQUEST, RESPONSE> HttpServerAttributesExtractorBuilder<REQUEST,
 
   @Override
   public void onStart(AttributesBuilder attributes, Context parentContext, REQUEST request) {
-    super.onStart(attributes, parentContext, request);
```

Comment:
Only the attributes that are added at the start will be available for sampling. Moving all the attributes to the `onEnd` will make sampling impossible.

Reply:
So I need to figure out which attributes impact sampling, and those attributes are the minimum set need to retain, right?

According to semantic-conventions(e.g. [http-spans](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#http-server-semantic-conventions)), I'd like to extract the unique attributes of span at the start without needing to pass them on. The common attributes and metrics attributes should be extracted at the end.